src/Copy/BrandCopy.elm

```diff
@@ -121,12 +121,12 @@ brandCopy key =
 
         HomeP1 ->
             CopySection
-                [ CopyText "When something goes wrong and you feel like the NHS has let you down it can be hard. You have a right to complain and the NHS needs your feedback in order to improve their service. Call The Advocacy People for free information about getting your voice heard and other organisations that can help."
-                , CopyText "If there are parts of your complaint you cannot do for yourself, you can ask for one of the The Advocacy People team to be your advocate. They will listen to you, support you to speak for yourself, and help you to make choices. They can also write letters for you, help you in complaint meetings, and talk to other people for you if necessary."
+                [ CopyText "When something goes wrong and you feel like the NHS has let you down, it can be hard. You have a right to complain and the NHS needs your feedback in order to improve their service. Call The Advocacy People for free information about getting your voice heard and other organisations that can help."
+                , CopyText "If there are parts of your complaint you cannot do for yourself, you can ask for one of The Advocacy People's team to be your advocate. They will listen to you, support you to speak for yourself, and help you to make choices. They can also write letters for you, help you in complaint meetings, and talk to other people for you if necessary."
```

Comment:
Not sure if it should be `'s` Does their own website give any clues?

Reply:
From a readability standpoint, I would advocate for this since it makes the sentence "flow" better. Just my two cents.

Reply:
I'm ok with it being written either way to be honest, so happy to keep Jess's suggestion if she feels the sentence flows better this way. 

I have seen many charities write it both ways.